assets/src/components/common/MathJaxFormula.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useCallback } from 'react';
+import { sanitizeMathML } from '../../utils/mathmlSanitizer';
+
+/**
```

Comment:
A good place to test that these work is a the hints section. Without the additional rendering logic here, markup appeared instead of the rendered formula when revealing new hints.
![hints](https://user-images.githubusercontent.com/333265/172911208-91e1899a-7202-4a68-8e4c-ee0d9e341090.gif)